holoviews/plotting/bokeh/renderer.py

```diff
@@ -28,22 +29,38 @@ class BokehRenderer(Renderer):
         Output render format for static figures. If None, no figure
         rendering will occur. """)
 
+    holomap = param.ObjectSelector(default='auto',
+                                   objects=['widgets', 'scrubber', 'server',
+                                            None, 'auto'], doc="""
+        Output render multi-frame (typically animated) format. If
+        None, no multi-frame rendering will occur.""")
+
+    mode = param.ObjectSelector(default='default',
+                                objects=['default', 'server'], doc="""
+         Whether to render the DynamicMap in regular or server mode. """)
```

Comment:
Is it only rendering DynamicMaps? Might want a bit more to say what server mode is about ...